### ISSUE TITLE
HTTP429 ERROR

### ISSUE THREAD

**Issue by koxtuichi** [ijjk/notion-blog#63]
Recently, when I view the list of blogs 10 times in a row (using the incognito window), I get HTTP429 error in Notion. And you will not be able to retrieve the articles. Is this a limitation on the Notion side?

**Comment by Lonor** [comment]
Yes. I've noticed it too. It seems like the Notion API now been protected by this rate limit. The segment of its response: 
```json
{
    "errorId": "84f42356-xxxx-4aa5-a20f-edcad0b601cc",
    "name": "UserRateLimitResponse",
    "message": "Please try again later.",
    "clientData": {
        "type": "rate_limited"
    }
}
```

**Comment by koxtuichi** [comment]
@Lonor thank you comment. And I also get same error message.

**Comment by Lonor** [comment]
Sorry, I don't know how to deal with this because the API was restricted by the Notion team. Hope the official API (#53) comes out earlier.

**Comment by koxtuichi** [comment]
Yes, I hope so too.
